client/app/templates/components/milestones/public-hearing.hbs

```diff
@@ -8,12 +8,14 @@
   {{~moment-format @hearing.disposition.dcpDateofpublichearing "LLLL"~}}
 </small>
 
-<div class="text-tiny" id="{{center-content action.dcpName}}">
+<div class="text-tiny">
   {{#each @hearing.hearingActions as |action index|}}
     {{#if action.dcpName}}
-      <span class="label light-gray tiny-margin-bottom tiny-margin-right" data-test-hearing-actions-list="{{@hearing.disposition.id}}{{index}}">
-        {{action.dcpName}}
-        <small>{{action.dcpUlurpnumber}}</small>
+      <span id="{{center-content action.dcpName}}">
```

Comment:
`<span class="{{if (is-text-lengthy action.dcpName) 'center-content'}}">`
OR
`<span class="{{if (is-text-lengthy action.dcpName 22) 'center-content'}}">`

Few things here: 
1. the span should use `class`, not `id` because there will be multiple <span> tags in the page using the `center-content` style. 
2. To make the code more expressive and readable, the helper should be focused to determining whether a text string is over a certain length. 
Then, the template will use that helper to determine whether or not to render the center-content class.

`{{center-content acation.dcpName}}` indicates very little to a developer about its intentions. what does center-content return? why is action.dcpName passed into center-content? 

By renaming the helper to something like `is-text-lengthy`, we indicate the purpose is to determine the length of an input text. It likely returns a boolean. With this boolean, we determine how the span should be styled.

Reply:
I renamed and refactored the helper method, and everywhere it was used (changed to a move obvious IF statement.